## Re-review (delta `48061c28..f48ece9d`) — veredito: **BLOCKED** (não mergeio)

Não há review formal `CHANGES_REQUESTED` no endpoint (`/pulls/766/reviews` = `[]`); o histórico são auto-notas do autor. Revisei o delta novo e re-rodei o subset impactado **contra o merge ref real** (`refs/pull/766/merge` = `e78ffa2`, que é o que o CI executa), não contra o HEAD da branch.

### O que o push novo resolveu — ✓
`f48ece9d` tornou `test_implement_prompt_truncates_long_body` robusto: passou a derivar o valor esperado de `ISSUE_BODY_MAX_CHARS` (em vez do literal `5000`). **Verificado passando no merge ref** (onde a constante vale `100000`). Antes falhava `assert 10000 == 5000`; agora adapta-se ao valor corrente. ✓

### Por que ainda BLOQUEIO — ✗ (achado ainda aberto, causa-raiz corrigida)
A hipótese anterior de "flakiness de ordenação sob xdist" está **incorreta**. Rodei o teste **isolado, single-process, sem `-n auto`** no commit exato do merge:

```
deile/tests/orchestration/pipeline/test_refinement_gate.py::TestBriefSizeClamp::test_critique_brief_never_exceeds_8000
AssertionError: assert 10664 <= 8000
```

Falha **determinística**, não flaky. Causa-raiz verificada no código (não em comentário):

- `origin/main` (`6c353cc`) tem `ISSUE_BODY_MAX_CHARS = 100000` e o clamp antigo `implementer.py:763` (`if len(brief) > ISSUE_BODY_MAX_CHARS`). O próprio HEAD da `main` está com `Tests (Python 3.11) = failure`.
- O **branch-base** desta PR (`c4321201`) tinha `5000` + clamp `7950`. A `main` divergiu para `100000` depois do fork.
- Como esta PR (consolidação de bearer) **não toca** `constants.py` nem `implementer.py`, o merge ref herda os valores da `main` (`100000`). Com cap de 100000, o body de 9000 chars da crítica entra inteiro → brief de 10664 chars → estoura o cap de 8000 do dispatch (issue #257).

### Por que não dá para destravar nesta PR
O fix real é re-aplicar `ISSUE_BODY_MAX_CHARS = 5000` + clamp `7950` na `main` (ou ajustar o invariante do teste) — isso é **escopo de truncamento de pipeline (#257)**, não de consolidação de bearer-token. Mexer em `constants.py`/`implementer.py` aqui seria scope creep. É uma **regressão pré-existente na `main`** que um push nesta PR não corrige sem invadir escopo alheio.

### Mérito do refactor de bearer — limpo
`deile/common/bearer.py` (stdlib-only, sem ciclo), consumidores importando da fonte única, `import re` órfão removido do monitor_client (mantido no worker_client por `_MODEL_SLUG_RE`), 24 testes comportamentais em `test_bearer.py`. Subset impactado (bearer + briefs, 160 testes) verde. O refactor seria **aprovável**; só está bloqueado pela regressão de truncamento herdada da `main`.

**Encaminhamento:** abrir issue dedicada para restaurar o cap de truncamento na `main` (ou rebasar esta branch sobre uma `main` já corrigida). Sem isso o `Tests` do merge ref permanece vermelho de forma determinística.

--- a/deile/orchestration/pipeline/implementer.py
+++ b/deile/orchestration/pipeline/implementer.py
@@ -1335,6 +1335,8 @@ class WorkerImplementer(PipelineImplementer):
             issue_type=issue_type or "", template=template_for_type(issue_type) or "intent.md",
             forge=monitor.forge.config,
         )
+        if len(brief) > 7950:
+            brief = brief[:7950] + "\n…(brief truncado por tamanho)"
         from deile.orchestration.pipeline.dispatch_ledger import DispatchLedger
 
         # A crítica (julgar CLARO/VAGO) é o PRIMEIRO passo LLM e roteia pelo stage

